src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -482,16 +481,20 @@ private static StripeResponse _rawRequest(
 			APIConnectionException, CardException, APIException {
 		StripeResponse response = _rawRequest(method, url, params, type, options);
 
-		int rCode = response.getResponseCode();
-		String rBody = response.getResponseBody();
-		String requestId = response.getRequestId();
+		int rCode = response.code();
+		String rBody = response.body();
+		String requestId = response.requestId();
```

Comment:
So I put a bit of thought behind this after @brandur-stripe's suggestions. My rationale for breaking with the `get`/`set` logic is that a `response` is generally immutable. You can't really change much about what you got back from the wire right?

So I think we should go with these methods because they're more familiar and they don't imply that there's some sort of mutability in the response.

Reply:
👍  This sounds perfectly reasonable.